src/modules/test/RecipeInvoker.cpp

```diff
@@ -96,6 +96,8 @@ void RecipeInvoker::TestBody()
         std::cout << "Waiting for " << m_recipe.m_waitSeconds << " seconds" << std::endl;
         std::this_thread::sleep_for(std::chrono::seconds(m_recipe.m_waitSeconds));
     }
+
+    std::remove("/etc/osconfig/osconfig_commandrunner.cache");
```

Comment:
Is it possible to save a copy of this file before running the tests, then restore the cache instead of removing it completely?

Reply:
Small corner case to consider that could impact a dev environment significantly: 

1. Send Reboot/Shutdown via the IoTHub
2. Stop OSConfig (agent/platform) 
3. Run `modulestest`
4. Restart OSConfig **<-** *this will cause your dev machine to Reboot/Shutdown*

Reply:
yes, it would affect it if that scenario was run. But this is a temporary workaround until we have a good setup/teardown solution

Reply:
This is now done in main, please remove from this PR. Do not merge with this in, will duplicate the change (the one in main is on a different line)

Reply:
Removed this in the latest iteration.